modules/tutorials/pages/create-couchbase-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+= Create and Deploy a Cluster with Capella Free Tier
+:description: Learn how to create and deploy a cluster using Capella Operational Free Tier.
+:page-topic-type: tutorial
+:page-pagination: full
+:page-toclevels: 2
+:navtitle: Create and Deploy a Cluster
+
+[abstract]
+{description}
+
+== Prerequisites
+
+Before starting this tutorial, you must have a Couchbase Capella account. 
+If you do not have one already, xref:cloud:get-started:create-account.adoc[Create an Account]. 
+
+== Create and Deploy a Cluster
+
+Every Capella account includes one free tier operational cluster. 
+In this tutorial, you will use the free tier cluster to create and manage student records. 
+
+To create and deploy a cluster: 
+
+. Sign in to https://cloud.couchbase.com/[Couchbase Capella]. 
+
+. On the *Operational Clusters* page, click btn:[Create Cluster].
+
+. Select *My First Project* as the project for your cluster. 
```

Comment:
Need to allow for the case where they've already created a project for whatever reason

```suggestion
. Select *My First Project* as the project for your cluster.
(If you've already created a project, you can select that instead.)
```